linkerd/core/src/main/scala/io/buoyant/linkerd/FailureAccrualInitializer.scala

```diff
@@ -16,7 +16,7 @@ abstract class FailureAccrualConfig extends PolymorphicConfig {
   var backoff: Option[BackoffConfig] = None
 
   @JsonIgnore
-  val backoffOrDefault =
+  lazy val backoffOrDefault =
```

Comment:
I think that –  if we're not in the habit of mutating these configs post-parsing – ideally, it would be nice if the config was immutable, and was instantiated by the parser with all the values set, rather than instantiated and then mutated by the parser (this error could have been avoided entirely!). I'm not sure if our parsing code would easily allow that in this instance, though...